.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+### 🗒 Description
+
+If helpful for the review process, please describe your changes from a technical or non technical perspective
+
+#
+### 🎫 Git ticket reference
```

Comment:
The ticket can be attached using measures from Github itself

Reply:
Measures in question

<img width="400" alt="image" src="https://github.com/user-attachments/assets/84e8b42e-d8c6-442b-b5e0-3f5e1328fdcb">

Reply:
Sounds good to me, if we add this as a check

Reply:
Haha, I didn't see this option in the side panel. I think I do not look often on the right of the issue.
I personally perfer to link them inline as text with key words like closes or closed. So I liked this part :)